### ISSUE TITLE
Implement the Mission Pillar Section

### ISSUE THREAD

**Issue by orama254** [reactdeveloperske/reactdevske-website#80]
### Depends On

- Pillar Component #79 

### What feature are you adding?

Mission Pillar Section

### Requirements

- Create a component named `MissionPillars` in the `components` folder that implements the screenshots below.
- It should make use of the existing `Pillar` component.
- Obtain the images from the Figma design. Use [THIS VIDEO](https://www.youtube.com/watch?v=T3rUlKf40M0) as a guide
- The images should use the Next Image component.
- Include the component in `index.page.tsx`

**SCREENSHOTS**

**Desktop**
![mission-component](https://user-images.githubusercontent.com/50571688/194320163-528136bd-cb8a-4950-8995-6e94abcb234b.png)

**Mobile**

![mission-component-mobile](https://user-images.githubusercontent.com/50571688/194678615-a4f6840d-2b55-43e1-b9a1-7cbcd32a7ee0.png)

[Figma Design](https://www.figma.com/file/TVwnaDhBGeVdnVKdf6H91C/React-developers-community-website?node-id=741%3A976)

### Acceptance Criteria

- [ ] The implementation should match the design - text content, typography, capitalization and spacing.
- [ ] The UI should be responsive and work well on both desktop and mobile viewports.

**Comment by Lekipising** [comment]
Requesting to be assigned this.

**Comment by orama254** [comment]
Hey @Lekipising 

Go ahead, remember to go through our [contribution guidelines](https://github.com/reactdeveloperske/reactdevske-website/blob/main/CONTRIBUTING.md) and also evaluate the acceptance criteria before submitting a pull request.

Happy hacking!!!

**Comment by orama254** [comment]
@Lekipising 

Also note that this issue depends on #79 . Check it out an let me know if you are ok implementing it or can wait for it to be implemented.

**Comment by Lekipising** [comment]
I will be fine with that. Please assign me

**Comment by orama254** [comment]
Alright, assigning you in a bit.

**Comment by Lekipising** [comment]
Hey @orama254 can I send the mission pillars in 1 PR together with the necessary implementation for #79?

**Comment by antosan** [comment]
> Hey @orama254 can I send the mission pillars in 1 PR together with the necessary implementation for #79?

@Lekipising Kindly implement each ticket in its own PR. First, send in a PR for #79 and once that is accepted and merged, then you can send in a separate PR for this ticket.

**Comment by Lekipising** [comment]
Thanks @antosan

**Comment by AJ-Kulundu** [comment]
@orama254 I want to be assigned this issue

**Comment by orama254** [comment]
Hey @AJ-Kulundu  

Go ahead, remember to go through our [contribution guidelines](https://github.com/reactdeveloperske/reactdevske-website/blob/main/CONTRIBUTING.md) and also evaluate the acceptance criteria before submitting a pull request.

Happy hacking!!!
